docs/user_guide/assessment/perform_fairness_assessment.rst

```diff
@@ -199,6 +199,11 @@ These are accessed through the :attr:`MetricFrame.by_group` property:
 
 All of these values can be checked against the original arrays above.
 
+Note that :class:`MetricFrame` is intended for analyzing the disparities 
```

Comment:
Shall we put this in a `.. note`? (Although I'm never sure whether those blocks are more noticeable or simply ignored by readers, haha.)